### ISSUE TITLE
Not working with multiple selection

### ISSUE THREAD

**Issue by danillos** [atombender/sublime_text_alternative_autocompletion#6]
Autocomplete not work with multiple selections.

**Comment by atombender** [comment]
Yeah, that's a documented limitation. That said — autocompletion with multiple selections will never work very well if you map the autocompletion to `escape`, since that's the key used to abort a multiselection. It will only be appropriate when mapping autocompletion to something else.
